communication/best-practices.md

```diff
@@ -0,0 +1,82 @@
+# Mailing list and Github usage best practice                                                       
+                                                                                
+The Kubernetes Mailing list or Google Groups functions as the primary means of  
+asynchronous communication for the project's                                    
+[Special Interest Groups (SIG)][sig-list] and [Working Groups (WG)][sig-list].  
+That's why you may want to set your filters in your email account to attain a
+good signal-to-noise ratio with regards to the mailing list messages and Github
+notifications. All the steps below are basically for Gmail users.
+
+Note: If you are looking to create and manage lists as a chair or other organizer, 
+check this mailing list [guideline][mailinglist-guidelines] doc.
+
+- [Example for setting filters to Kubernets Mailing lists](#example-for-setting-filters-to-kubernets-mailing-lists)
+- [Examples for setting filters to Kubernetes Github notifications](#examples-for-setting-filters-to-kubernetes-github-notifications)
+                                                                                
+## Example for setting filters to Kubernets Mailing lists           
+
+It depends on the SIG or/and WG you are involved in, you would like to setup
+your filters in your Gmail account, so you don't need to have all emails
+structred in one place.
+Create a filter following the procedure below:
+
+- In your Gmail account click on **Settings**:
+  - **Filters and Blocked Addresses** -> Scroll down and click **create a new filter**
+  - In the **to** fields write the email of the SIG's Google Group. 
+  - **Create filter** -> Check the box ** Apply the label** and create new
+label by choosing **New label...** in the dropdown list.
+  - Click on the **Create filter**.
+- You can also use filters directly for lists.
+  - E.g: Matches: list:"kubernetes-dev@googlegroups.com"
+  - Do this: Apply label "lists/kubernetes-dev"
+
+## Examples for setting filters to Kubernetes Github notifications
+
+In order to not get spammed by Github notifications and to get all the 
+notifications in the right place, here is an example of filters to create/apply
+on your Gmail account:
+
+- Stick a blue label on anything kubernetes-related:
```

Comment:
Nothing in this config dictates color, I just happen to use blue :)

Reply:
yes, but it is good to show that you can setup filters and categorize them by color.. That's why I kept them